lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/src/main.c

```diff
@@ -18,8 +18,22 @@
 
 #include "stdlib/stats/base/dists/triangular/mgf.h"
 #include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/expm1.h"
 #include "stdlib/math/base/special/exp.h"
-#include "stdlib/math/base/special/pow.h"
+
+/**
+* Helper function for repeated computation in the MGF formula.
+*
+* @private
+* @param x   input value
+* @return    evaluated result
+*/
+static double phi2( const double x ) {
```

Comment:
This, however, is fine and does **not** need to be split out into a separate file.